### DISCUSSION TITLE
How to control the padding in a table of contents on a sidebar

### DISCUSSION THREAD

**Discussion by WalidGharianiEAGLE** [alshedivat/al-folio#1472] [Q&A] [answered]
The table of content sidebar is taking too much space, is there any way to control the padding of the table of content sidebar?

![Screenshot from 2023-06-09 08-24-48](https://github.com/alshedivat/al-folio/assets/57763453/e8150801-cae3-4236-a798-9e2cde0593f1)

**Comment by IamShubhamGupto** [comment, marked as answer]
I think you adjust that using `_config.yml`
```
# Dimensions
max_width: 800px
```
